### PR TITLE
Enable multiple path for hdfs storage and instructions

### PR DESCRIPTION
**Pull request by yatharthranjan**
Allows for extending the capacity of hadoop

### CODE REVIEW

File: dcompose-stack/radar-cp-hadoop-stack/README.md [hunk 1]
```diff
@@ -195,7 +195,53 @@ You can check the logs of CRON by typing `grep CRON /var/log/syslog`.
 
 ### HDFS
 
-This folder contains useful scripts to manage the extraction of data from HDFS in the RADAR-base Platform.
+#### Advanced Tuning
+
+To increase the amount of storage you can add multiple paths as destinations for data storage as follows -
```
Comment (afolarin) [inline dcompose-stack/radar-cp-hadoop-stack/README.md]:
```suggestion
To increase the amount of storage horizontally you can add multiple paths as destinations for data storage as follows -
```

File: dcompose-stack/radar-cp-hadoop-stack/README.md [hunk 2]
```diff
@@ -195,7 +195,53 @@ You can check the logs of CRON by typing `grep CRON /var/log/syslog`.
...
+
+- Add the required paths as environment variables in `.env` file similar to the other hdfs paths like -
```
Comment (afolarin) [inline dcompose-stack/radar-cp-hadoop-stack/README.md]:
```suggestion
- Add the required paths as environment variables in `.env` file similar to the other hdfs paths like 
 HDFS_DATA_DIR_<NODE#>_<VOLUME#> -
```

**Comment by blootsvoets** [comment]
Do these additional directories actually count as replicas? For the name node, data is duplicated across the listed directories by simply writing all data twice. This increases the redundancy of the system, but I cannot see in the documentation that it increases the number of replicas. That would mean that using this technique, the amount of data that needs to be stored is doubled. In other projects we have just increased the number of data nodes to spread the data over more volumes.

**Comment by yatharthranjan** [comment]
These will not mean more replicas. This is just for increasing the storage capacity of existing data nodes(replicas). We just did this in production as our disks were full so we added new disks and hence had to add additional storage path to data nodes. 

> That would mean that using this technique, the amount of data that needs to be stored is doubled
Not Sure i understand this but this is just for adding a new volume as a storage to datanodes

**Comment by afolarin** [comment]
I think the data nodes and name nodes treat volume addition differently.

**Pull request merged**